lib/timers.js

```diff
@@ -250,15 +250,6 @@ function listOnTimeout() {
 
     var domain = timer.domain;
     if (domain) {
-
-      // If the timer callback throws and the
-      // domain or uncaughtException handler ignore the exception,
-      // other timers that expire on this tick should still run.
-      //
-      // https://github.com/nodejs/node-v0.x-archive/issues/2631
-      if (domain._disposed)
-        continue;
```

Comment:
hah. so disposal has never worked for `setImmediate()`. let's use that as a demonstration of how little it must have been used.